test/presubmit-tests.sh

```diff
@@ -25,6 +25,13 @@
 # in a net-negative contributor experience.
 export DISABLE_MD_LINTING=1
 
+# Download and install GO 1.12rc1, which is required for websocket.
+# TODO(tcnghia): Move this to test-infra.
+echo "> Installing Go 1.12"
+go get golang.org/dl/go1.12rc1
+go1.12rc1 download
+cp $GOPATH/bin/go1.12rc1 $(which go)
```

Comment:
I'd like to know how we fix this.  cc @chaodaiG

Reply:
Should golang update be independent of the test CL?